azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -280,6 +280,20 @@ public ExecutableFlow fetchExecutableFlow(final int execId) throws ExecutorManag
     }
   }
 
+  public void unsetExecutorIdForExecution(final int executionId) throws ExecutorManagerException {
```

Comment:
Please add some comments for the method.

Reply:
added